app/graphql/query.ts

```diff
@@ -97,6 +97,15 @@ export const balanceUsd = (client: ApolloClient<unknown>): number =>
 export const balanceBtc = (client: ApolloClient<unknown>): number =>
   _.find(getWallet(client), { id: "BTC" }).balance
 
+export const getWalletNetworkOnly = async (
```

Comment:
instead of having a `getWalletNetworkOnly`, I'll recommend a `getWallet` function where you can pass a `fetchPolicy` to override the default settings.